src/Stratis.Bitcoin.Features.BlockStore/Models/BlockByHashQueryModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Stratis.Bitcoin.Features.BlockStore.Models
+{
+    public abstract class QueryModelBase
+    {
+        public bool OutputJson { get; set; }
```

Comment:
Json should really be the default. Maybe `ShowAsHex`can be better? What do you think?